src/picongpu/include/simulation_defines/param/bremsstrahlung.param

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright 2016 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+namespace picongpu
+{
+namespace particles
+{
+namespace bremsstrahlung
+{
+
+/** enable bremsstrahlung */
+#ifndef ENABLE_BREMSSTRAHLUNG
+#define ENABLE_BREMSSTRAHLUNG 0
+#endif
+
+/** ion related params
+ */
+namespace ion
+{
+    /** atomic number of the target ion species */
+    BOOST_CONSTEXPR_OR_CONST uint32_t TARGET_Z = 29;
```

Comment:
This is actually an attribute of the ion species - see [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/src/picongpu/include/particles/traits/GetAtomicNumbers.hpp).